docs/core/porting/breaking-changes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Breaking changes
+description: Breaking changes between releases of .NET and how it impacts compatibility.
+author: stebon
+ms.date: 06/14/2021
+---
+# Breaking Changes
+
+Changes that affect compatibility, otherwise known as breaking changes, will occur between versions of .NET. This is particularly impactful when porting from .NET Framework to .NET due to certain technologies not being available as well as the changes needed to make .NET a cross-platform framework.
+
+We strive to maintain a high level of compatibility between .NET versions, so while breaking changes will occur, they are carefully considered.
+
+Before upgrading major versions check the breaking changes documentation for changes that might impact you.
+
+## Categories of breaking changes.
+
+There are several types of breaking changes...
+
+- modifications to the public contract
+- behavioral changes
+- Platform support
+- Internal implementation changes
+- Code changes
+
+For details about what is allowed or disallowed see [Changes that affect compatibility](../compatibility/index.md)
+
+## Types of compatibility.
+
+Compatibility refers to the ability to compile or execute code on a version of a .NET implementation other than the one with which the code was originally developed.
```

Comment:
Style guide info related to the word `execute`: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33512
```suggestion
Compatibility refers to the ability to compile or run code on a version of a .NET implementation other than the one with which the code was originally developed.
```